bundles/org.openhab.binding.digiplex/src/main/resources/ESH-INF/thing/area.xml

```diff
@@ -23,7 +23,7 @@
 		</channels>
 		<config-description>
 			<parameter name="refreshPeriod" type="integer">
-				<label>Refresh time of area status (in seconds)</label>
+				<label>Refresh Time of Area Status (in Seconds)</label>
```

Comment:
The `(in Second)` should be completely removed from the label. It can be mentioned in the description and even better by adding the `unit=` attribute to the parameter.